app/src/main/res/layout/mission_item.xml

```diff
@@ -37,8 +37,13 @@
 				android:layout_alignParentRight="true"
 				android:layout_centerVertical="true"
 				android:layout_marginRight="1dp"
+
+				android:src="?options"
```

Comment:
I restarted the travis build and the strange exoplayer bug is gone. Unfortunaltely, the build still fails because of this merge issue. There are two `src` attributes for the imageView.